### PR TITLE
changeable hud

### PR DESCRIPTION
**Pull request by 15360enes**
if you turn on logover it will render logo with version crazy

### CODE REVIEW

File: src/main/java/me/earth/earthhack/impl/modules/client/hud/HUD.java [hunk 1]
```diff
@@ -122,8 +127,8 @@ public HUD() {
     }
 
     protected void renderLogo() {
-        if (logo.getValue()) {
-            renderText("3arthh4ck - " + Earthhack.VERSION, 2, 2);
+        if (logo.getValue() || logover.getValue()) {
```
Comment (3arthqu4ke) [inline src/main/java/me/earth/earthhack/impl/modules/client/hud/HUD.java:130]:
```java
if (logo.getValue()) {
```
instead pls.

**Review by 3arthqu4ke — CHANGES_REQUESTED** [review]

**Comment by 15360enes** [comment]
I dont know how github works how do I implement this to my pull req

**Comment by 3arthqu4ke** [comment]
Just make another commit where you fix it, the PR will be updated.

**Comment by 15360enes** [comment]
I think i did it

**Review by 3arthqu4ke — APPROVED** [review]

**Pull request closed without merging**